projects/components/src/time-range/time-range.component.scss

```diff
@@ -42,12 +56,43 @@
 
 .refresh {
   margin-left: 8px;
+
+  ::ng-deep {
```

Comment:
+1 on cleaning up the styling here.

Reply:
was able to remove 2 instances of it below however not the first on L62. There's a miss match in height of buttons and popover selectors. All the selectors on the page headers are 34px, while our buttons are either 32 or 36. Normally this wouldn't be much of an issue but since the selector and button are right next to one another it's easy to see the height difference.

Reply:
I guess that's fine. We should make a note of it somewhere possibly create a tech debt so that we can fix it later